<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectToLocalWidget</class>
 <widget class="QWidget" name="ConnectToLocalWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>51</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="mainFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="accessibleName">
         <string>ConnectToLocal</string>
        </property>
        <property name="text">
         <string>Local profiling</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="contentFrame">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="startOrbitServiceButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Start OrbitService</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="statusLabel">
           <property name="text">
            <string>connection status placeholder</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../icons/orbiticons.qrc"/>
 </resources>
 <connections/>
</ui>
